### PR TITLE
Improve empty table in list views

### PR DESCRIPTION
**Pull request by aljesusg**
Fix https://github.com/kiali/kiali/issues/1674

## Workloads list
workloads
## Applications list
applications
## Services list
services
## Istio list
istio

### CODE REVIEW

**Comment by prachiyadav** [comment]
Screenshot from 2020-02-13 01-08-04
Screenshot from 2020-02-13 01-08-07
Screenshot from 2020-02-13 01-08-13

File: src/pages/AppList/AppListComponent.tsx [hunk 1]
```diff
@@ -143,6 +144,15 @@ class AppListComponent extends FilterComponent.Component<AppListComponentProps,
           ]}
         />
       </VirtualList>
+    ) : (
+      <EmptyState variant={EmptyStateVariant.full}>
```
Comment (xeviknal) [inline src/pages/AppList/AppListComponent.tsx]: What do you think of moving this emptystate to a separate component? The behavior between other list pages are really similar.

File: src/pages/AppList/AppListComponent.tsx [hunk 2]
```diff
@@ -143,6 +144,15 @@ class AppListComponent extends FilterComponent.Component<AppListComponentProps,
...
+        <Title headingLevel="h5" size="lg">
+          No Applications found
+        </Title>
+        <EmptyStateBody>
+          No applications in namespaces : {this.props.activeNamespaces.map(ns => ns.name).join(', ')}
```
Comment (xeviknal) [inline src/pages/AppList/AppListComponent.tsx]: Nitpick:
The space between `namespaces` and `:` is a bit annoying, isn't it?
Besides that, what do you think to change the copy depending on the namespace count: `No applications in default namespace` or `No applications in namespaces: blah, blah, blah`

**Comment by aljesusg** [comment]
## 1 namespaces
1n

## multiple namespaces
ns

**Pull request merged**